src/outputs/gb_sas.py

```diff
@@ -65,7 +65,7 @@ def output_gb_sas(
     # Map to the CORA statuses from the statusencoded column
     df = map_o.create_cora_status_col(df, cora_mapper)
 
-    # Map the sizebands based on frozen employment
+    # Map the sizebands based on  employment
```

Comment:
Again, small thing, but we've taken out frozen here - should we take it out from the function description in `map_output_cols.py` and where it's called in `outputs\short_form.py` and `outputs\tau.py`?

Reply:
I couldn't find what you were referring to here. But I found another mention of "frozen" in `map_output_cols.py`. Could you clarify where this is?

Reply:
In map_output_cols, the frozen employment is correct, no need to change.

Reply:
Please change the comment in tau.py, line 67: # # Map the sizebands based on column "employment"